### PR TITLE
Docs for Tabs Component

### PR DESCRIPTION
**Pull request by vintagepenguin**
### MX-Components-PR: https://github.com/mxenabled/mx-react-components/pull/395

screen shot 2016-08-08 at 10 26 17 am

### CODE REVIEW

**Comment by paniclater** [comment]
:shipit:

File: components/Changelog.js [hunk 1]
```diff
@@ -6,6 +6,12 @@ const Changelog = React.createClass({
       <div>
         <h1>Change Log</h1>
 
+        <h3>4.2.21</h3>
+        <ul>
+          <li>Tabs Component - add it and make it work for mobile (<a href='https://github.com/mxenabled/mx-react-components/pull/395'>#395</a>)</li>
```
Comment (cerinman) [inline components/Changelog.js]: The wording for this is odd.  How about something like `Adds new Tabs component.  See docs for details.`

Comment (vintagepenguin) [inline components/Changelog.js]: ✔️

**Comment by cerinman** [comment]
:shipit:

**Pull request merged**